modules/apps/object/object-test/src/testFunctional/tests/tests/CreateObject.testcase

```diff
@@ -79,12 +79,31 @@ definition {
 	}
 
 	@description = "LPS-135549 - Verify it is possible to create a Date field"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanCreateDateField {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.openObjectAdmin();
 
-		// TODO LPS-138819 CanCreateDateField pending implementation
+		ObjectPortlet.selectCustomObject(name = "CustomObject");
 
+		Click(locator1 = "ObjectPortlet#OBJECT_FIELDS_TAB");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Not Required Field",
+			fieldName = "date",
+			fieldType = "Date",
+			isRequired = "false",
+			objectName = "CustomObject");
```

Comment:
Since for this test we need to validate that it is possible to create the field, you need to add the field via UI instead of the API.